repl/convert_ast.py

```diff
@@ -7,19 +7,68 @@
 import qwerty_ast
 #################### COMMON CODE FOR BOTH @QPU AND @CLASSICAL DSLs ####################
 
+# quick fix, going to create another file most likely for errors
+class QwertySyntaxError(Exception):
+    pass
+
 def convert_to_qwerty(py_ast):
     print(ast.dump(py_ast, indent=4))
```

Comment:
It's good to have this line for debugging, but we should either remove it or put it behind some kind of debug flag. A Qwerty programmer would be pretty confused if they typed in a Qwerty expression and got a Python AST printed out on their screen